base/db/R/get.trait.data.pft.R

```diff
@@ -197,14 +197,14 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon, trait.names,
           } else if (length(trait.data.check) == 0) {
             PEcAn.logger::logger.warn("New and existing trait data are both empty. Skipping this check.")
           } else {
-            current_traits <- dplyr::bind_rows(trait.data.check, .id = "trait") %>%
-              dplyr::select(-mean, -stat)
-            existing_traits <- dplyr::bind_rows(existing_trait_data, .id = "trait") %>%
-              dplyr::select(-mean, -stat)
+            current_traits <- dplyr::bind_rows(trait.data.check, .id = ".data$trait") %>%
+              dplyr::select(-mean, -.data$stat)
+            existing_traits <- dplyr::bind_rows(existing_trait_data, .id = ".data$trait") %>%
+              dplyr::select(-mean, -.data$stat)
```

Comment:
Because this is inside a string, I don't think the `.data` pronoun is needed (or will even work) here.

```suggestion
            current_traits <- dplyr::bind_rows(trait.data.check, .id = "trait") %>%
              dplyr::select(-mean, -.data$stat)
            existing_traits <- dplyr::bind_rows(existing_trait_data, .id = "trait") %>%
              dplyr::select(-mean, -.data$stat)
```